front/src/styles/scss/applications/stdcm/_results.scss

```
@@ -90,6 +102,13 @@
               margin-left: 8px;
               color: rgb(60, 202, 128);
             }
+
+            .alternative-simulation {
+              display: flex;
+              align-items: center;
+              margin-left: 8px;
```

only 4px on the mockup
```suggestion
              margin-left: 4px;
```